addons/mail/static/src/core/persona_service.js

```diff
@@ -55,6 +55,16 @@ export class PersonaService {
             this.store.registeredImStatusPartners?.push(persona.id);
         }
     }
+
+    /**
+     * @returns {[integer]}
+     */
+    getMostRecentChatPartnerIds() {
+        return Object.values(this.store.threads)
+            .filter((thread) => thread.type === "chat")
+            .sort((a, b) => b.lastInterestDateTime.ts - a.lastInterestDateTime.ts)
+            .map((thread) => thread.chatPartnerId);
```

Comment:
`lastInterestDateTime` on `Thread` model could be `undefined`. So should take that into account